src/Illuminate/Contracts/Concurrency/Driver.php

```diff
@@ -3,7 +3,7 @@
 namespace Illuminate\Contracts\Concurrency;
 
 use Closure;
-use Illuminate\Foundation\Defer\DeferredCallback;
+use Illuminate\Concurrency\Defer\DeferredCallback;
```

Comment:
@taylorotwell This might be a breaking change but as Concurrency is still in beta we might allow it.